config/application.rb

```diff
@@ -23,6 +23,7 @@ class Application < Rails::Application
     # -- all .rb files in that directory are automatically loaded.
 
     config.autoload_paths << Rails.root.join("app", "uploaders")
+    config.autoload_paths << Rails.root.join("app", "decorators")
```

Comment:
Yep. I can use call `GameDecorator` and any uploader in Rail's console without those lines. No problem 😎